lib/stripe/event.ex

```diff
@@ -15,6 +15,7 @@ defmodule Stripe.Event do
     livemode: false,
     pending_webhooks: 0,
     request: nil,
+    status: nil",
```

Comment:
I think there is an extra " here.

Reply:
whoops. I blame the GitHub web interface